sandbox/settings.py

```diff
@@ -0,0 +1,206 @@
+"""
+Django settings for sandbox project.
+
+Generated by 'django-admin startproject' using Django 2.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.2/ref/settings/
+"""
+
+import os
+
+from configurations import Configuration, values
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# pylint: disable = W0232
```

Comment:
It would be wonderful to add a comment explaining why you choose to silent this pylint warning. :pray:

Reply:
Done :slightly_smiling_face: